sentences/en/_common.yaml

```diff
@@ -376,15 +376,15 @@ expansion_rules:
   are_any: "[<are> ]<any>"
   how_many: "how many[ of the]"
   brightness: "{brightness}[[ ]%| percent]"
-  light: "[the ](light|lights|lighting|lamp|lamps)"
-  turn: "(turn|switch|change)"
+  light: "[the ](light|lights|lighting|lamp|lamps|bulb|bulbs|light fixture|light fixtures|light switch|light switches)"
```

Comment:
I said this before: bulbs, LEDs, chandeliers etc. are not good words to include because it would be weird if a user would say "turn on the bulb" in an area with only LEDs and no light bulbs (or one light bulb and many other lights), then all the lights would unexpectedly turn on. It would work anyway if the user would add the name/alias "bulb" to what they know is an actual light bulb.

Also, if we really want to include it, at least let's make it `[light] bulb`. I really, really don't want to include it, though.

Reply:
Yeah, I agree

Reply:
We discussed this on Discord, so yea. I'll drop this one. I think it makes sense to remove given that bulb, fixture, and switch are too specific and not used generally to refer to "lights".